### ISSUE TITLE
bash_profile damaged by installer

### ISSUE THREAD

**Issue by BishopGIS** [nextgis/nextgis_installer#29]
The source .bash_profile
```
# Setting PATH for Python 3.5
# The original version is saved in .bash_profile.pysave
PATH="/Library/Frameworks/Python.framework/Versions/3.5/bin:${PATH}"

# Setting PATH for Python 3.6
# The original version is saved in .bash_profile.pysave
PATH="/Library/Frameworks/Python.framework/Versions/3.6/bin:${PATH}"
export PATH
export SSL_CERT_FILE=/Users/username/Applications/NextGIS/Library/Frameworks/openssl.framework/Resources/ssl/certs/cert.pem
export CURL_CA_BUNDLE=/Users/username/Applications/NextGIS/Library/Frameworks/openssl.framework/Resources/ssl/certs/cert.pem
export PROJ_LIB=/Users/username/Applications/NextGIS/Library/Frameworks/proj.framework/Resources/proj
export GDAL_DATA=/Users/username/Applications/NextGIS/Library/Frameworks/gdal.framework/Resources/data
#export QCA_PLUGIN_PATH=/Users/username/Applications/NextGIS/Library/Plugins/Qt4/
```

After install:
```
# Setting PATH for Python 3.5
# The original version is saved in .bash_profile.pysave
PATH="/Library/Frameworks/Python.framework/Versions/3.5/bin:${PATH}"
export PATH:/Users/username/Applications/NextGIS/usr/bin

# Setting PATH for Python 3.6
# The original version is saved in .bash_profile.pysave
PATH="/Library/Frameworks/Python.framework/Versions/3.6/bin:${PATH}"
export PATH:$PATH
export SSL_CERT_FILE=/Users/username/Applications/NextGIS/Library/Frameworks/openssl.framework/Resources/ssl/certs/cert.pem
export CURL_CA_BUNDLE=/Users/username/Applications/NextGIS/Library/Frameworks/openssl.framework/Resources/ssl/certs/cert.pem
export PROJ_LIB=/Users/username/Applications/NextGIS/Library/Frameworks/proj.framework/Resources/proj
export GDAL_DATA=/Users/username/Applications/NextGIS/Library/Frameworks/gdal.framework/Resources/data
#export QCA_PLUGIN_PATH=/Users/username/Applications/NextGIS/Library/Plugins/Qt4/
```

**Comment by MikhanGusev** [comment]
Maybe make our own environment for starting all NextGIS apps like it was implemented for Windows?

**Comment by BishopGIS** [comment]
As I understand users - the some kind of our shell is extremely unusual. So let's fix the bug. It seems to me current code search `export PATH=`

But in this case we have only `export PATH`

**Comment by BishopGIS** [comment]
Also installer mast create backup file on first install. State of bash_profile without and add by installer. Let's name it `.bash_profile.ngbackup`

**Comment by BishopGIS** [comment]
Fixed in d03f41ed66a7dbc958243a0e04399c829cb872e5
